packages/fxa-shared/db/config.ts

```diff
@@ -261,5 +261,37 @@ export function makeRedisConfig() {
         doc: 'Minimum connection count for the subhub responses Redis pool',
       },
     },
+
+    customs: {
+      enabled: {
+        default: false,
+        doc: 'Enable Redis for customs server rate limiting',
+        format: Boolean,
+        env: 'CUSTOMS_REDIS_ENABLED',
+      },
+      host: {
+        default: 'localhost',
+        env: 'CUSTOMS_REDIS_HOST',
+        format: String,
+      },
+      port: {
+        default: 6379,
+        env: 'CUSTOMS_REDIS_PORT',
+        format: 'port',
+      },
+      password: {
+        default: '',
+        env: 'REDIS_PASSWORD',
```

Comment:
should this have a CUSTOMS_ prefix?

Reply:
I changed the stage/prod config to use both env vars, and filed https://github.com/mozilla/fxa/pull/16516 to change this name